tests/Resources/Page/OCPDashboard/Pods/Pods.robot

```diff
@@ -1,17 +1,18 @@
 *** Settings ***
 Resource   ../../OCPDashboard/Page.robot
 Resource   ../../ODH/ODHDashboard/ODHDashboard.robot
-Library         ../../../../libs/Helpers.py
+Library    ../../../../libs/Helpers.py
 
 *** Keywords ***
-Get Pod Logs
+Get Pod Logs Manual Version
```

Comment:
What do you mean by manual version?

Reply:
I wanted to change it to avoid conflicts with the Openshift CLI lib. I called it "manual" because in this version the robot is going to "read" the logs by clicking in the openshift UI

Reply:
Then perhaps `Get Pod Logs From UI` would be a better name?

Reply:
that's the one! I buy it :) i will change it, thanks

Reply:
done in this commit 889625840993aa438c1a7d82f0ffa8fe10dc1298